### ISSUE TITLE
Option to ignore PIN style passwords

### ISSUE THREAD

**Issue by ccoenen** [andrew-schofield/keepass2-haveibeenpwned#72]
I keep a list of classic "PIN" numbers in KeePass as well. They are obviously very weak and all of them are flagged every time. I have no control over these entries (for the most part), and I am aware that 4-digt numbers are baaaad passwords.

But because the list will always return 20 entries, I might miss one or two actual passwords that are _actually_ in danger.

**Comment by strayge** [comment]
Last update allows to check just specific group.
You can move PIN password to another group and don't check it.

It's not enough?

**Comment by Sanaki** [comment]
I'd say the easy solution would be to respect a specific advanced string field (hibp_nocheck = true or similar) when running checks. In my case I'd like to exempt all instances of "admin", "root", "webmaster", and other common and non-personal usernames.

**Comment by ccoenen** [comment]
good idea! That's probably more useful than limiting it to 4-digit PIN numbers. One _might_ even add a button to the list "ignore this entry from now on" which would add the custom field.

**Comment by andrew-schofield** [comment]
This sounds like a duplicate of #43 now I think?

**Comment by ccoenen** [comment]
Yes, pretty much.
